app/qml/project/MMProjectHomePage.qml

```diff
@@ -0,0 +1,93 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import lc 1.0
+
+import "../components"
+import "../misc"
+import "../inputs"
+import "."
+
+Item {
+  id: root
+
+  property int projectModelType: ProjectsModel.EmptyProjectsModel
```

Comment:
this is always ProjectsModel.LocalProjectsModel for home page

it doesn't need to be property at all, just set it in the ProjectList down